apollo-router/src/spec/query.rs

```diff
@@ -150,6 +150,7 @@ impl Query {
         variables: Object,
         schema: &Schema,
     ) -> Vec<Path> {
+        println!("Formatting: {response:?}");
```

Comment:
```suggestion
```

Reply:
Sorry about that. I've removed it and rebased (and updated to the new changelog thingy).